spec/features/service_providers_spec.rb

```diff
@@ -257,7 +257,7 @@
       choose('service_provider_identity_protocol_openid_connect_private_key_jwt', allow_label_click: true)
 
       saml_attributes =
-        %w[acs_url assertion_consumer_logout_service_url sp_initiated_login_url return_to_sp_url block_encryption signed_response_message_requested]
```

Comment:
should we add some sort of positive test that checks that the field is on the page in both cases?